### DISCUSSION TITLE
Flink 结合 Light Schema Change 实现 DDL 同步

### DISCUSSION THREAD

**Discussion by gtk96** [apache/doris#14226] [A - General / Q&A] [answered]
公众号文章关于这个功能的代码 demo 有吗，对这个功能很感兴趣

**Comment by LemonLiTree** [comment, marked as answer]
https://github.com/apache/doris-flink-connector/blob/schemachange-1.14/flink-doris-connector/src/test/java/org/apache/doris/flink/CDCSchemaChangeExample.java
后面会补充到官网文档
